### DISCUSSION TITLE
When I turn off TV LEDs stays on

### DISCUSSION THREAD

**Discussion by mkaflowski** [awawa-dev/HyperHDR#531] [General]
When I turn off TV LEDs stays on with the last one configuration and app WLED keeps saying that hyperion is sending data. Why and how to fix it?

**Comment by awawa-dev** [comment]
Probably HyperHDR: is not notified that the TV is going to power off (hacked webos lacks proper power management messaging?) or it is killed before it has a chance to send closing message to the WLED or the TV network is already down when it tries to do that? Webos is not officially supported and I cant test HyperHDR using this hacked OS. You can try to switch HyperHDR to UDP Raw driver, it doesn't send/require a 'goodbye' message to WLED.

**Comment by mkaflowski** [comment]
Factory reset for TV fix that.

**Comment by oywino** [comment]
I too have this "problem". I use WLED and HyperHDR 19. My TV is a Samsung QN95B  How can the TV and WLED be aware of one another? Neither has any settings or config to enable them to find the other? Does the TV by default broadcast some kind of standard "Googbye" signal that WLED can catch?

**Reply by awawa-dev** [reply]
I doubt it's the same problem, on the webos the HyperHDR process can be killed without a proper shutdown procedure that affects WLED driver (but doesn't affect simpler WLED udpraw alternative driver). But you've provided too little detail about your setup, symptoms, and exactly what situation they occur in, what WLED reports and what is in HyperHDR logs, to make any more judgments.

**Reply by oywino** [reply]
> I doubt it's the same problem, on the webos the HyperHDR process can be killed .....
Please excuse my ignorance, but do Samsung TV's use WebOS ?   I thought they used Tizen.... ?
